bundle/manifests/operator.antrea.vmware.com_antreainstalls.yaml

```diff
@@ -1,8 +1,8 @@
-apiVersion: apiextensions.k8s.io/v1beta1
+apiVersion: apiextensions.k8s.io/v1
```

Comment:
The change  in this CR is due to the update from v1beta1 to v1

Reply:
Should the `deploy/operator.antrea.vmware.com_antreainstalls_crd.yaml` be updated to be consistent with this file?